### PR TITLE
initial document for HTMLAnchorElement.hreflang

### PR DESCRIPTION
**Pull request by ShubhamOulkar**
This adds the missing HTMLAnchorElement.hreflang page.

### resource link
[hreflang](https://html.spec.whatwg.org/multipage/links.html#attr-hyperlink-hreflang)

Part of the project to document all interoperable features ([mdn/discussions#476](https://github.com/orgs/mdn/discussions/476))

### CODE REVIEW

File: files/en-us/web/api/htmlanchorelement/hreflang/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+title: "HTMLAnchorElement: hreflang property"
+short-title: hreflang
+slug: Web/API/HTMLAnchorElement/hreflang
+page-type: web-api-instance-property
+browser-compat: api.HTMLAnchorElement.hreflang
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`hreflang`** property of the {{domxref("HTMLAnchorElement")}} interface contains a string that is the language of the linked resource.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/hreflang/index.md:11]:
```suggestion
The **`hreflang`** property of the {{domxref("HTMLAnchorElement")}} interface is a string that is the language of the linked resource.
```

File: files/en-us/web/api/htmlanchorelement/hreflang/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+It reflects the `hreflang` attribute of the {{HTMLElement("a")}} element and is the empty string (`""`) if there are none.
+
+Web browsers and search engines may use this information to better understand the language of the linked content, but they are not required to follow it. The value provided for the `hreflang` attribute must adhere to format defined in {{RFC(5646, "Tags for Identifying Languages (also known as BCP 47)")}}.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/hreflang/index.md:15]:
```suggestion
Web browsers and search engines may use this information to understand the language of the linked content better, but they are not required to follow it. The value provided for the `hreflang` attribute adheres to the format defined in {{RFC(5646, "Tags for Identifying Languages (also known as BCP 47)")}}. If not, it is ignored.
```

File: files/en-us/web/api/htmlanchorelement/hreflang/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Despite the presence of the `hreflang` attribute, web browsers are instructed not to rely solely on this when fetching the linked resource. Instead, they should use language information directly associated with the resource (e.g. through HTTP headers) to determine its language.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/hreflang/index.md:17]:
```suggestion
Web browsers do not rely solely on the `hreflang` attribute after fetching the linked resource. Instead, they use language information directly associated with the resource (e.g., through HTTP headers) to determine its language.
```

File: files/en-us/web/api/htmlanchorelement/hreflang/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+```html
+<a id="exampleLink" href="https://example.com" hreflang="en-IN">Example Link</a>
+```
+
+```js
+const anchorElement = document.getElementById("exampleLink");
+console.log(anchorElement.hreflang); // Outputs: "en-IN"
+```
+
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/hreflang/index.md:33]: I think we can quickly transform this into a live example. Can you do it, please?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thank you very much. I propose a few changes, but this is close to be ready to land.

**Comment by teoli2003** [comment]
Do you want to add it to the property list of `HTMLAnchorElement` in this PR or in separate one?

File: files/en-us/web/api/htmlanchorelement/hreflang/index.md (hunk 4 above)
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlanchorelement/hreflang/index.md:33]: ok I will do it.

**Comment by ShubhamOulkar** [comment]
@teoli2003 property is already listed on [HTMLAnchorElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement#instance_properties) page. 
I this only `ping` is not listed. we need small update in [google docs](https://docs.google.com/spreadsheets/d/1lvITVZJM8dx8s_Kee5UNI1hDfPNF0gXQwjPop5xIF5E/edit#gid=369545647)

File: files/en-us/web/api/htmlanchorelement/hreflang/index.md (hunk 4 above)
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlanchorelement/hreflang/index.md:42]: @teoli2003 live example is updated

**Comment by teoli2003** [comment]
(In the future, when a PR is ready for a re-review) click on the re-review button, so I get notified)
Capture d’écran 2024-01-06 à 08 13 14

**Review by teoli2003 — APPROVED** [review]
👍 Thank you very much!

**Comment by ShubhamOulkar** [comment]
> (In the future, when a PR is ready for a re-review) click on the re-review button, so I get notified) Capture d’écran 2024-01-06 à 08 13 14

sure @teoli2003.

**Comment by ShubhamOulkar** [comment]
> 👍 Thank you very much!

**Pull request merged**